<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detect from file(s)</string>
  </property>
  <widget class="QLabel" name="original_image">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>416</width>
     <height>416</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Original Image</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="result_image">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>416</width>
     <height>416</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Result</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="file_path_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select Image File Path...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SelcPathBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="detect_btn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>200</y>
     <width>51</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Go</string>
   </property>
  </widget>
  <widget class="QLabel" name="savepath_label">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>470</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select Result Saving Path...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SavePathBtn">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="segment_save">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>520</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Segment_Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_cap">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>520</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Cap</string>
   </property>
  </widget>
  <widget class="QLabel" name="notification">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Notification</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
